modules/ROOT/pages/con_pipeline-cluster-resource.adoc

```diff
@@ -0,0 +1,48 @@
+[]
+= Cluster Resource
+
```

Comment:
Do we need two spaces between the title and the content? We usually use only one space in EAP documents.